vfs/dir.go

```diff
@@ -187,6 +188,21 @@ func (d *Dir) _readDir() error {
 		return err
 	}
 
+	err = d._readDirFromEntries(entries, nil, time.Time{})
+	if err != nil {
+		return err
+	}
+
+	d.read = when
+	return nil
+}
+
+func (d *Dir) _readDirFromDirTree(dirTree walk.DirTree, when time.Time) error {
```

Comment:
This and the function below should have a "must be called with the lock held" commend like _readDir

Reply:
Right, I missed that comment at _readDir.
Documentation is updated.